node/service/src/service.rs

```diff
@@ -370,41 +376,31 @@ async fn start_node_impl(
 
 /// Build the import queue for the parachain runtime.
 #[cfg(not(feature = "frequency-no-relay"))]
+/// Build the import queue for the parachain runtime.
```

Comment:
One of these is not needed:

```suggestion
#[cfg(not(feature = "frequency-no-relay"))]
/// Build the import queue for the parachain runtime.
```

Reply:
Nice catch! I’ll update this in my follow-up PR to avoid triggering the CI again.